.github/workflows/build.yml

```diff
@@ -43,6 +43,20 @@ jobs:
             cxx-compiler: clang++
             cc-compiler: clang
             compiler-desc: Clang
+          - name_prefix: Linux CentOS 7 VFX CY2023
+            release_prefix: linux-vfx2023
+            ostype: linux
+            distro: centos7
```

Comment:
The 2023 aswf-docker containers are based on Rocky Linux 8 (8.8 as of the time they were published). Looks like the only reference to the `distro` symbol is to check whether it is ubuntu or not, which really seems to mean whether the build is in a `aswf-docker` container or not. Changing this value to `rocky8` would work, but perhaps the symbol should be called `aswfdockerbuild` and be a boolean?

Reply:
Fixed.